servers/account-data-deleter/src/queueHandlers/batchDeleteHandler.ts

```diff
@@ -0,0 +1,174 @@
+import { EventEmitter } from 'events';
```

Comment:
The unique logic was pulled out to this class and separated from the abstract class QueueHandler.ts